metron-analytics/metron-profiler-common/src/main/java/org/apache/metron/profiler/DefaultMessageRouter.java

```diff
@@ -53,10 +56,16 @@
    */
   private StellarStatefulExecutor executor;
 
+  /**
+   * Responsible for creating the {@link Clock}.
+   */
+  private ClockFactory clockFactory;
```

Comment:
The `ClockFactory` helps with the timestamp logic.  This used to live in the `ProfileSplitterBolt` as that is where the timestamp extract logic lived.  Now the `MessageRouter` does this for all Profiler ports.